src/libraries/System.Diagnostics.Process/src/Microsoft/Win32/SafeHandles/SafeProcessHandle.Unix.cs

```diff
@@ -24,7 +24,7 @@ public sealed partial class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvali
         // Process.{Safe}Handle to initalize and use a WaitHandle to successfully use it on
         // Unix as well to wait for the process to complete.
 
-        private readonly SafeWaitHandle _handle;
+        private readonly SafeWaitHandle? _handle;
```

Comment:
NIT: Seems all constructors setting it with `SetHandle(existingHandle);` helper, which we normally leave as non null and initialize with `null!`

Reply:
This is a partial class, and the constructors defined in `SafeProcessHandle.cs` that doesn't set that particular field.

Reply:
yes it is partial class, i meant constructors in SafeProcessHandle calling `SetHandle(existingHandle);` mehtod to set it (chain of constructors calling the constructor which calling this method)

Reply:
Ah, you're right. Changing now.